solidity/contracts/BeaconBackportRewards.sol

```diff
@@ -0,0 +1,141 @@
+pragma solidity ^0.5.17;
```

Comment:
Do we want to add KEEP ASCII art to the new contracts?

Reply:
Yes we do :grin: